src/pmpo_fortran.f90

```diff
@@ -453,29 +449,32 @@ subroutine polympo_getMeshVtxRotLat(mpMesh, nVertices, latitude) &
   !> @param mpmesh(in/out) MPMesh object
   !> @param nComps(in) number of components, should always be 2
   !> @param nVertices(in) numVertices
-  !> @param array(in) input vertices velocity 2D array (2,numVtx)
+  !> @param uVel(in) vertices u-component velocity 1D array (numVtx)
+  !> @param vVel(in) vertices v-component velocity 1D array (numVtx)
   !---------------------------------------------------------------------------
-  subroutine polympo_setMeshVel(mpMesh, nComps, nVertices, array) &
+  subroutine polympo_setMeshVel(mpMesh, nComps, nVertices, uVel, vVel) &
```

Comment:
we do not need 'nComps' (see polympo_setMeshVtxCoords)

Reply:
removed https://github.com/SCOREC/polyMPO/commit/8bec689310aa9db8a2ecd8673a002858a0506d28